src/main/kotlin/org/jitsi/jibri/sipgateway/pjsua/PjsuaClient.kt

```diff
@@ -66,7 +66,9 @@ class PjsuaClient(private val pjsuaClientParams: PjsuaClientParams) : SipClient(
                 "--id", "${pjsuaClientParams.sipClientParams.displayName} <sip:jibri@127.0.0.1>",
                 "--config-file", CONFIG_FILE_LOCATION,
                 "--log-file", "/tmp/pjsua.out",
-                "sip:${pjsuaClientParams.sipClientParams.sipAddress}"
+                "--max-calls=1",
+                if (pjsuaClientParams.sipClientParams.autoAnswer)
+                    "--auto-answer=200" else "sip:${pjsuaClientParams.sipClientParams.sipAddress}"
```

Comment:
am i  reading this right that this  will put it  in a mode  where, instead of calling it out it will go into a mode where it waits for  an incoming call?  is the  `200` some kind of timeout?

Reply:
200 is the sip code for the auto-answer:
```
--auto-answer=code  Automatically answer incoming calls with code (e.g. 200)
```
Yes, the idea is not to trigger this functionality from the xmpp side, but from the sip side for incoming sip calls.

Reply:
so is there some kind  of timeout?  will jibri  just sit like this forever?

Reply:
Good point I will add a timeout, thanks.